cli/options/options.go

```diff
@@ -73,6 +73,13 @@ var ConfigFile = cli.StringFlag{
 	Usage: "path to the node configuration file (overrides --config-path option)",
 }
 
+// RelativePath is a flag for commands that use node configuration and provides
+// the path to the consensus wallet instead of reading it from the config file.
+var RelativePath = cli.StringFlag{
+	Name:  "relative-path",
```

Comment:
s/`relative-path`/`consensus-wallet`?

With the current implementation it may be an absolute path, and it's OK. And the CLI flag should be more common, because when we start a node with some `neo-go node -t --relative-path ./smth.json` it's not very clear what's the purpose of this `--relative-path`, especially when we have `--config-path` and `--config-file` options.

Reply:
I kept it to how Roman named it in https://github.com/nspcc-dev/neo-go/issues/3179#issuecomment-1785154680 but I agree that the current name isn't very descriptive.

Reply:
Sorry, I'm still not clear enough with this. "relative-path" should be added to _any_ relative path in the config. You may have other wallets for other services for example. Or you may want to have a relative path for your DB. Overriding a single parameter is not very exciting.

Reply:
And in that sense it'd be nice to add the same flag to `cli/vm/vm.go` that uses config files as well.

The result is that any non-absolute path in the config is `relative-path + the_value_from_the_config`. I think it'd be more powerful this way.

Reply:
I'm going to be way too inefficient in figuring out where all those paths are and how they work so I think it's best to leave it up to you guys then.